opm/simulators/wells/RateConverter.hpp

```diff
@@ -432,6 +433,8 @@ namespace Opm {
             template <typename ElementContext, class EbosSimulator>
             void defineState(const EbosSimulator& simulator)
             {
+                if (!active)
+                    return;
```

Comment:
Yaaah - the *Say no to superfluous braces* club has just accepted a new sturdy member!

Reply:
weell. this was not really a grand political statement from me, more of a habitual mistake haha.